tests/test_notifications.py

```diff
@@ -1116,6 +1116,12 @@ def test_get_settings_url_for_user(self):
         url = emails.get_settings_url(self.user._id, self.user)
         assert_equal(url, web_url_for('user_notifications', _absolute=True))
 
+    def test_get_notifications(self):
```

Comment:
The function being tested is not related to the OSF notifications system. Can you move this to `TestConfigureMailingListViews` in `tests/test_views.py`?